File: javascript/3342.js
var input = require('fs').readFileSync('./dev/stdin', 'utf8');
var lines = input.split(' ');
var s = parseInt(lines.shift());
var m = Math.pow(s, 2);
var brancas = m%2 === 0 ? m / 2 : Math.ceil(m / 2);
var pretas = m%2 === 0 ? m / 2 : Math.floor(m / 2);
console.log(brancas + " casas brancas e " + pretas + " casas pretas");

File: python/3342.py
import math
v=int(input())
m=pow(v,2)
if m%2 == 0:
    b=m/2
    p=m/2
else:
    b=math.ceil(m / 2)
    p=math.floor(m/2)
print(str(int(b))+" casas brancas e "+str(int(p))+" casas pretas")